packages/create-react-native-library/templates/common/README.md

```diff
@@ -4,9 +4,15 @@
 
 ## Installation
 
+<% if (project.nitro) { -%>
+```sh
+npm install <%- project.slug %> react-native-nitro-modules
```

Comment:
Probably should also add a very short description and link to nitro website below this so people know what is the `react-native-nitro-modules` package that they are installing.

Reply:
I added:
> `react-native-nitro-modules` is required as this library relies on [Nitro Modules](https://nitro.margelo.com/).